src/client/opamLockCommand.ml

```diff
@@ -184,7 +188,11 @@ let lock_opam ?(only_direct=false) st opam =
         |> map_of_set (`other_dep typ)
         |> OpamPackage.Map.union (fun _v _o -> `other_dep typ) depends_map
     else
-      (OpamConsole.msg "Not all dependencies are satisfied, won't include them\n";
+      (OpamConsole.msg "Not all dependencies of %s are satisfied, not \
+                        including these: %s\n"
```

Comment:
The error message is confusing. I’m not sure what it means

Reply:
`"These dependencies are not satisfied, they will be suppressed from the .opam.locked file"` ?

Reply:
What does that mean/will do?

Reply:
Accually I understand it now. For some reason the first time i read it, i didn’t understand the meaning of the sentance.